src/components/AppRouter/AppRouter.js

```diff
@@ -1,15 +1,14 @@
-import PropTypes from 'prop-types'
 import React from 'react'
 import { Route, Switch } from 'react-router-dom'
-import Home from '../../pages/home/Home'
+import Overview from '../../pages/Overview/Overview'
 import { sections } from '../../pages/sections.conf'
```

Comment:
Unrelated, but since you're tackling the removal of patterns that are hard to work with in our codebases, I think that these .conf files are also a strange pattern for a react app. We could also put that on the todo list. What do you think?

Reply:
I've filed https://jira.dhis2.org/browse/DHIS2-11343 (as well as https://jira.dhis2.org/browse/DHIS2-11344 for the data administration app)

Reply:
👍